app.env.example

```diff
@@ -69,3 +69,8 @@ FIREBASE_SERVER_KEY=
 
 # Grafana Dashboard
 DASHBOARD_URL=http://localhost:8000/login/generic_oauth
+GF_ADMIN_USERNAME=admin
+GF_ADMIN_PASSWORD=password
+GF_BASE_URL=http://localhost:8000
+DATASOURCE_URL=localhost:5432
```

Comment:
@hengsokly better use the term GF_DATASOURCE_URL, GF_DATASOURCE_USER, and GF_DATASOURCE_USER_PASSWORD, include GF_ prefix to state and include another variable GF_DATASOURCE_USER as they can connect outside of default admin datasource account(postgres for PG or root for MySQL)